### PR TITLE
AC-836 Improved UI for add provider screen

### PR DESCRIPTION
**Pull request by rishabh-997**
## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-836

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by rishabh-997** [comment]
WhatsApp Image 2020-08-14 at 2 33 56 AM

**Comment by LuGO0** [comment]
@rishabh-997  could you remove the underline fromt the edit text It would look better then.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/providermanagerdashboard/addprovider/AddProviderActivity.java [hunk 1]
```diff
@@ -53,14 +53,4 @@ protected void onCreate(Bundle savedInstanceState) {
...
-    @Override
-    public void showNoInternetConnectionSnackbar() {
-        mSnackbar = Snackbar.make(addProviderFragment.doneFAB,
-            getString(R.string.no_internet_connection_message), Snackbar.LENGTH_INDEFINITE);
-        View sbView = mSnackbar.getView();
-        TextView textView = sbView.findViewById(com.google.android.material.R.id.snackbar_text);
-        textView.setTextColor(Color.WHITE);
-        mSnackbar.show();
-    }
 }
```
Comment (LuGO0) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/providermanagerdashboard/addprovider/AddProviderActivity.java:56]: @rishabh-997 is there a specific reason why you removed this snackbar ?

Comment (rishabh-997) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/providermanagerdashboard/addprovider/AddProviderActivity.java:56]: Yes, we needed to override this method in order to move the floating button up when snack bar is shown. Now I have removed the floating action button, so need to use this.

**Comment by rishabh-997** [comment]
> @rishabh-997 could you remove the underline from the edit text as you have done in addpatients

Actually I am using `FilledBox` style instead of `OutlinedBox` style, and this underlines properties are by default present in materials design so I did not change it in order to comply with their guidelines.

**Pull request merged**